google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1220,6 +1225,12 @@ public Builder disableLeaderAwareRouting() {
       return this;
     }
 
+    @BetaApi
+    public Builder disableDirectPath() {
```

Comment:
* Instead of a method, wouldn't it be better to use an internal system property? AFAIU, we only require this for our test infrastructure right now. 
* Also, my understanding is customers would never want to disable directpath, given they don't need to understand about internal proxy connections between GCP and Borg. If yes, that also pushes us to not expose a public method and instead use a system property?